### DISCUSSION TITLE
Add app erro after upgrading from version 10.3.5 to version 11.0.1

### DISCUSSION THREAD

**Discussion by boanergescosta** [gravitational/teleport#18256] [Q&A]
Hi everybody!!!

After an upgrade from version 10.0.5 to version 11.0.1 I can not add new apps to teleport anymore, in version 11 the auth_servers attribut has changed to proxy_server, but when I generate the application token with the command "tctl tokens add --type=app --ttl=1h" the token is created using auth-server attribut, but as I said before, the version 3 of teleport config file uses proxy_server attribut and the system throug the following error message:

ERROR: configuration error
initialization failed
config: cannot use both auth_server and proxy_server

Any idea on how to fix this problem??

Best regards

**Comment by zmb3** [comment]
Can you share your `teleport.yaml`? (make sure to redact any sensitive values)

**Comment by webvictim** [comment]
It sounds like the `tctl tokens add` output is probably buggy and needs to be updated, but it's definitely still possible to make this work. As @zmb3 said please share your `teleport.yaml` file and we can advise!
